mvpa2/algorithms/searchlight_hyperalignment.py

```diff
@@ -0,0 +1,476 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the PyMVPA package for the
+#   copyright and license terms.
+#
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""Searchlight-based hyperalignment"""
+import numpy as np
+from mvpa2.base.state import ClassWithCollections
+from mvpa2.base.param import Parameter
+from mvpa2.algorithms.hyperalignment import Hyperalignment
+from mvpa2.measures.base import Measure
+from mvpa2.datasets import Dataset, vstack
+from mvpa2.mappers.staticprojection import StaticProjectionMapper
+from scipy.linalg import LinAlgError
+from mvpa2.measures.searchlight import sphere_searchlight
+from mvpa2.mappers.zscore import zscore
+from scipy.sparse import coo_matrix, dok_matrix, csc_matrix
+from mvpa2.featsel.helpers import FractionTailSelector, FixedNElementTailSelector
+
+if __debug__:
+    from mvpa2.base import debug
+
+def compute_feature_scores(datasets, exclude_from_model=None):
```

Comment:
this function name is so non-descriptive... what about `across_dataset_consistency_score`. Also needs explicit statement that data should be standardized/zscored to actually yield "sum of maximal correlations" per each voxel